aries_cloudagent/wallet/sd_jwt.py

```diff
@@ -0,0 +1,320 @@
+"""Operations supporting SD-JWT creation and verification."""
+
+import re
+from typing import Any, List, Mapping, Optional, Union
+from marshmallow import fields
+from jsonpath_ng.ext import parse
```

Comment:
I had to look up what `parse` was referring to, it might be better to use an alias here (`jsonpath_parse` for instance).